contracts/lockdrop/src/contract.rs

```diff
@@ -2260,7 +2300,7 @@ fn update_user_lockup_positions_and_calc_rewards(
             LOCKUP_INFO.save(deps.storage, lockup_key, &lockup_info)?;
         };
 
-        lockup_astro_rewards = lockup_info.astro_rewards;
+        let lockup_astro_rewards = lockup_info.astro_rewards;
 
         // Save updated Lockup state
         total_astro_rewards += lockup_astro_rewards;
```

Comment:
Do we need to add checked function here as well?

Reply:
will be useful. Done, thanks.